packages/react-examples/src/react-switch/Switch/Switch.stories.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
```

Comment:
You should probably remove this file since we are collocating the stories within their own packages.